src/targets/node/nodeTarget.ts

```diff
@@ -161,6 +162,10 @@ export class NodeTarget implements Target {
     return this._cdp;
   }
 
+  public async afterBind() {
+    this._cdp.resume();
```

Comment:
I don't see where afterBind is called from

Reply:
fixed :)